hardware_interface/doc/hardware_components_userdoc.rst

```diff
@@ -4,7 +4,7 @@ Hardware Components
 -------------------
 Hardware components represent abstraction of physical hardware in ros2_control framework.
 There are three types of hardware Actuator, Sensor and System.
-For details on each type check `Hardware Components description <https://ros-controls.github.io/control.ros.org/getting_started.html#hardware-components>`_.
+For details on each type check `Hardware Components description <https://control.ros.org/master/doc/getting_started/getting_started.html#hardware-components>`_.
```

Comment:
```suggestion
For details on each type check :ref:`overview_hardware_components` description.
```
Within spinx you can use [cross-reference links](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations) to such labels:
https://github.com/ros-controls/control.ros.org/blob/311a61a8cd86b215468999fd518f57ad6339e773/doc/getting_started/getting_started.rst?plain=1#L79

Btw: there seem to be more broken links to ros-controls.github.io in other markdown files -> there we have to hardcode the published links.

Reply:
Thanks, that's obviously the better solution!